lang/java/ipc/src/test/java/org/apache/avro/TestProtocolHttps.java

```diff
@@ -48,14 +38,14 @@ public Server createServer(Responder testResponder) throws Exception {
     System.setProperty("javax.net.ssl.password", "avrotest");
     System.setProperty("javax.net.ssl.trustStore", "src/test/truststore");
     System.setProperty("javax.net.ssl.trustStorePassword", "avrotest");
```

Comment:
It might make sense to set the rest of the properties on the connector which were declared above, or remove those if they are not used/needed.

Reply:
I believe those system properties are also used by the client side portions (which I haven't touched) to setup the SSL stuff for the client that connect to it.